vitest/files.test.ts

```diff
@@ -28,40 +29,156 @@ describe('Tests', () => {
         parser.parse(fileReader, 37);
 
         //assert here
+        const errors = parser.getAllErrors();
+        const expectedError = new ErrorInformationRecord('0', '001', '0', '000000', '000000', '000', '000000', '000', 'RNS9308', 'T', '50', '057', 'Compilation stopped. Severity 20 errors found in program.');
+        expectedError.setFileName('/home/REINHARD/builds/hll/literalInMessage.pgm.rpgle');
+        assert.deepStrictEqual(errors, [expectedError]);
+
+        const fileIDRecords = parser.getAllFileIDRecords();
+        const expectedFileIDRecord = new FileIDRecord('0', '001', '000000', '60', '/home/REINHARD/bob-recursive-example/QSQLCSRC/ANZ_FILE2.SQLC', '20230429010835', '0');
+        assert.deepStrictEqual(fileIDRecords, [expectedFileIDRecord]);
+
+        const exceptions = parser.getException();
+        assert.strictEqual(exceptions.message, "Cannot read property 'getInputFileID' of undefined");
```

Comment:
I don't think we should be getting this error.  But I think Sanjula fixed this issue today.
Once he merges tomorrow morning let's test again.